### ISSUE TITLE
Bug - modulated parameter ranges modify disconnected parents

### ISSUE THREAD

**Issue by PowerUser64** [BespokeSynth/BespokeSynth#1517]
## Describe the bug

After a modulator is disconnected, newly-connected modulators can still modify the low and high values of the modulation.

## Build info

f82acc4f2fae30d5f2d85fdb3c90dd71e928f55b

## Steps To Reproduce

video

[co-modulation.webm](https://github.com/BespokeSynth/BespokeSynth/assets/24578572/91d4cfe7-968a-45dd-8894-a72cfe234650)

Note: this works with many types of modules.

## Expected vs. actual behavior

The low and high values should only change for the currently-connected modulator.

**Comment by PowerUser64** [comment]
you can even attach two modulators to the same parameter like this

[more co-modulation.webm](https://github.com/BespokeSynth/BespokeSynth/assets/24578572/9d05e231-16a6-4d0d-8382-ec943c22774d)

**Comment by merkys** [comment]
Is this still reproducible for you at current HEAD (11b88e396cc9c090cebfe2dcd81e0bcd40777bf6)? I do not seem to be able to reproduce it. Two modulators remain attached to `valuesetter`, but only the first one modulates it, the second one seems to be ignored.

**Comment by PowerUser64** [comment]
Yeah, I was able to reproduce it using the setup I used in the original video.

- pressuretocv -> valuesetter (in slider mode)
- swap to lfo -> valuesetter
- move LFO high/low values
- min/max values on pressuretocv change too

I think this is tied to the modulator having some kind of a range control.

https://github.com/user-attachments/assets/e7c245db-1d79-40a5-a5b9-e1585e9ce155
